packages/date/src/DateField.d.ts

```diff
@@ -10,6 +10,7 @@ export type DateFieldProps = {
   labelAttrs?: LabelProps;
   required?: boolean;
   helpId?: string;
+  placeholder?: string;
```

Comment:
Sorry, missed this before. This is extending the DateProps which already has the `placeholder` prop so we don't need to add it here